recipe/meta.yaml

```diff
@@ -158,11 +154,12 @@ outputs:
         - zstd
       run:
     files:
-      - lib/libdwarf.so.*        # [linux]
-      - lib/libdwarf.*.dylib     # [osx]
+      - lib/libdwarf.so          # [linux]
+      - lib/libdwarf.dylib       # [osx]
     test:
       commands:
-        - test ! -f $PREFIX/lib/libdwarf*.a
```

Comment:
Can we keep this test to ensure we aren't accidentally packaging the static lib?